scripts/install.tpl.sh

```diff
@@ -11,7 +11,7 @@ olm_version=$OLM_V0_VERSION
 cert_mgr_version=$CERT_MGR_VERSION
 rukpak_version=$RUKPAK_VERSION
 
-if [[ -z "$olm_version" || -z "$cert_mgr_version" || -z "$rukpak_version" ]]; then
+if [[ -z "$cert_mgr_version" || -z "$rukpak_version" ]]; then
     err="Error: Missing component version(s) for: "
     if [[ -z "$olm_version" ]]; then
```

Comment:
Lines 16 through 18 can also be removed since we aren't checking the `olm_version` anymore :)